src/main/java/cc/woverflow/hytils/handlers/chat/modules/triggers/AutoFriend.java

```diff
@@ -35,7 +35,7 @@ public class AutoFriend implements ChatReceiveModule {
     @Override
     public void onMessageReceived(@NotNull ClientChatReceivedEvent event) {
         String message = event.message.getUnformattedText().replace("\n", "");
```

Comment:
this line can be removed, doesnt seem like message is used anywhere. it should have been removed before but I just kept it in because I forgor 💀